src/vue/modules/verification/general-form/components/section-address.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="verification-general-form__section">
+    <p class="verification-general-form__section-label">
+      {{ 'general-form.address-details-lbl' | globalize }}
+    </p>
+
+    <div class="app__form-row">
+      <div class="app__form-field">
+        <input-field
+          white-autofill
+          v-model="line1"
+          @blur="touchField('line1')"
+          name="verification-general-address-line-1"
+          :label="'general-form.address-line-1-lbl' | globalize"
+          :error-message="getFieldErrorMessage('line1')"
+          :disabled="isDisabled"
+        />
+      </div>
+      <div class="app__form-field">
+        <input-field
+          white-autofill
+          v-model="line2"
+          @blur="touchField('line2')"
+          name="verification-general-address-line-2"
+          :label="'general-form.address-line-2-lbl' | globalize"
+          :error-message="getFieldErrorMessage('line2')"
+          :disabled="isDisabled"
+        />
+      </div>
+    </div>
+
+    <div class="app__form-row">
+      <div class="app__form-field">
+        <input-field
+          white-autofill
+          v-model="city"
+          @blur="touchField('city')"
+          name="verification-general-address-city"
+          :label="'general-form.address-city-lbl' | globalize"
+          :error-message="getFieldErrorMessage('city')"
+          :disabled="isDisabled"
+        />
+      </div>
+      <div class="app__form-field">
+        <input-field
+          white-autofill
+          v-model="state"
+          @blur="touchField('state')"
+          name="verification-general-address-state"
+          :label="'general-form.address-state-lbl' | globalize"
+          :error-message="getFieldErrorMessage('state')"
+          :disabled="isDisabled"
+        />
+      </div>
+    </div>
+
+    <div class="app__form-row">
+      <div class="app__form-field">
+        <input-field
+          white-autofill
+          v-model="postalCode"
+          @blur="touchField('postalCode')"
+          name="verification-general-address-postal-code"
+          :label="'general-form.address-postal-code-lbl' | globalize"
+          :error-message="getFieldErrorMessage('postalCode')"
+          :disabled="isDisabled"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import FormMixin from '@/vue/mixins/form.mixin'
+import SectionMixin from './section.mixin'
+import GetterAccessorMixin from './getter-accessor'
+
+import { mapMutations } from 'vuex'
+import { types } from '../store/types'
+import { required } from '@validators'
+
+export default {
+  name: 'section-address',
+  mixins: [FormMixin, SectionMixin, GetterAccessorMixin],
+  computed: {
+    line1: {
+      get () { return this.getter(types.line1) },
+      set (v) { this.setLine1(v) },
```

Comment:
`this.getter` is a nice thing but I do not understand why you do not use `mapGetters` instead? You use `mapActions`, but do not use `mapGetters` it breaks consistency and brings extra confuse here.
you can use the getters into a `storedLine1` to avoid the naming issue

if there is still a valuable reason of keeping the `this.getter`, pls move it to `section.mixin.js`